ecs/src/nox/ecs/component/Transform.cpp

```diff
@@ -0,0 +1,129 @@
+#include "Transform.h"
```

Comment:
Rather do: include <nox/ecs/components/Transform>, as we are not guaranteed that the files will be contained in the same folders.